.changeset/quick-islands-ring.md

```diff
@@ -0,0 +1,5 @@
+---
+"astro": patch
+---
+
+Prevent URLs in `content` attributes being escaped for multiple params to work
```

Comment:
```suggestion
Prevents URLs in attributes from being escaped in static builds
```
I think it concerns any attribute, eg. hrefs

Reply:
Yeah I think you're right. I've just set it as a general URL check for any attribute for now.